### DISCUSSION TITLE
Unhandled Rejection (TypeError) with yup validation.

### DISCUSSION THREAD

**Discussion by behzadmehrabi** [react-hook-form/react-hook-form#5791] [Q&A]
Hi, I have `fieldArray` named `items`, which has some complex object inside of it,
when I try to `trigger` validation manually for an index of array(e.g `await trigger('items.0')`),
I get this error: 
```Unhandled Rejection (TypeError): Cannot read property 'ref' of undefined```

Am I missing anything?

**Comment by bluebill1049** [comment]
can you share a codesandbox?
